orchagent/switchorch.cpp

```diff
@@ -26,6 +26,7 @@ const map<string, sai_switch_attr_t> switch_attribute_map =
     {"ecmp_hash_seed",                      SAI_SWITCH_ATTR_ECMP_DEFAULT_HASH_SEED},
     {"lag_hash_seed",                       SAI_SWITCH_ATTR_LAG_DEFAULT_HASH_SEED},
     {"fdb_aging_time",                      SAI_SWITCH_ATTR_FDB_AGING_TIME},
+    {"debug_shell_enable",                  SAI_SWITCH_ATTR_SWITCH_SHELL_ENABLE},
```

Comment:
once enabled, how do you disable?

Reply:
This is a create and set event as per SAI spec.[ ](https://github.com/opencomputeproject/SAI/blob/master/inc/saiswitch.h#L1671)

Reply:
https://github.com/opencomputeproject/SAI/blob/master/inc/saiswitch.h#L1671

Reply:
By setting 1 to 0 in below json, the debug shell can be disabled
```
[
    {
        "SWITCH_TABLE:switch": {
            "debug_shell_enable": "1"
        },
        "OP": "SET"
    }
]
```

Reply:
ok